### ISSUE TITLE
[Suggestion] Keep the edit behavior consistent with NERDTree single file move/copy's

### ISSUE THREAD

**Issue by sihunqu123** [PhilRunninger/nerdtree-visual-selection#10]
Thanks for your nice work on this useful plugin. Just a small suggestion, that the behavior of `Esc` doesn't' match the expectation of common software users.

## Issue
When move/copy using `nerdtree-visual-selection`, you'll get a prompt for inputting the target location, and 
+ the operation will continue(ask you if need to process this file one by one) if you press `Enter` when editing.
+ the operation won't get abort even I press `Esc`. U will still be asked whether need to process those files.

While for the original NERDTree single file move/copy:
+ the operation will continue if you press `Enter` when editing.
+ the operation will be aborted if you press `Esc` when editing.

## Expected behavior:
No more prompt/query if `Esc` pressed.

**Comment by PhilRunninger** [comment]
**NERDTree** uses the `input()` function to get the new file path, and pressing `Esc` causes it to quit immediately and return an empty string.

**nerdtree-visual-selection** uses the `confirm()` function to get the user's selection. Pressing `Esc` when it is asking for input seems to work differently. I needed to press it twice in order for Vim to recognize it. While the `Esc` key functionality has been implemented in #12, it might just be easier to use the `C` key at the `[Y]es, (N)o, (A)ll, (C)ancel:` prompt instead.
